tsfeatures/__init__.py

```diff
@@ -1,4 +1 @@
-#!/usr/bin/env python
-# coding: utf-8
-
-from .tsfeatures import *
```

Comment:
Please restore this below version

Reply:
added the line `from .tsfeatures import *` below the current `__version__` identifier